fsharp-backend/src/LibBackend/Tracing.fs

```diff
@@ -0,0 +1,247 @@
+/// Tracing for real execution
+module LibBackend.Tracing
+
+open FSharp.Control.Tasks
+open System.Threading.Tasks
+
+open Prelude
+open Tablecloth
+
+module PT = LibExecution.ProgramTypes
+module RT = LibExecution.RuntimeTypes
+module PT2RT = LibExecution.ProgramTypesToRuntimeTypes
+module AT = LibExecution.AnalysisTypes
+module Exe = LibExecution.Execution
+module Interpreter = LibExecution.Interpreter
+module DvalReprInternalDeprecated = LibExecution.DvalReprInternalDeprecated
+
+module LD = LibService.LaunchDarkly
+module Rollbar = LibService.Rollbar
+module Telemetry = LibService.Telemetry
+
+/// Tracing can go overboard, so use a per-handler feature flag to control it.
+/// sampling is disabled for a canvas, no traces will be saved. tlids will still be
+/// saved
```

Comment:
I don't understand this comment (the 2nd and third line).
Are these comments supposed to refer to the individual cases under `TraceSamplingRule.T`?